module/VuFind/src/VuFind/Controller/OverdriveController.php

```diff
@@ -99,11 +99,38 @@ public function mycontentAction()
                 $checkoutsUnavailable = true;
             } else {
                 foreach ($checkoutResults->data as $checkout) {
+                    $mycheckout = [];
```

Comment:
You could shave off a line of code by replacing these two lines with:

`$mycheckout = compact('checkout');`

That will initialize $mycheckout to ['checkout' => $checkout] for you.